modules/common/configmap/configmap.go

```diff
@@ -34,6 +34,23 @@ import (
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+
+
+// Hash function creates a hash of a ConfigMap's Data and BinaryData fields and
+// returns it as a safe encoded string.
+func Hash(configMap *corev1.ConfigMap) (string, error) {
```

Comment:
`Hash` does not return an error. It should probably return one if `configMap` is nil

Reply:
ahh `util.OpjectHash` does. maybe we check for configMap is nil

Reply:
other then that +2 from me

Reply:
Done